windows.js

```diff
@@ -4,8 +4,26 @@ import execall from 'execall';
 
 const TEN_MEBIBYTE = 1024 * 1024 * 10;
 
-export default async function killTabs() {
-	const command = 'wmic process where Caption=\'chrome.exe\' get CommandLine,ProcessId /format:list';
+export default async function killTabs(options) {
+	const browsers = []
+
+	if (options.chromium) {
+		browsers.push('Caption=\'chromium.exe\'')
+	}
+
+	if (options.chrome) {
+		browsers.push('Caption=\'chrome.exe\'')
+	}
+
+	if (options.brave) {
+		browsers.push('Caption=\'brave.exe\'')
+	}
+
+	if (options.edge) {
+		browsers.push('Caption=\'msedge.exe\'')
+	}
+
+	const command = `wmic process where "${browsers.join(' or ')}" get CommandLine,ProcessId /format:list`;
```

Comment:
It would be great if you could switch to `childProcess.execFile` while you're at it. That would simplify the quoting here.

Reply:
Something like the following?? (It is a working example)

```js
const args = ['process', 'where', browsers.join(' or '), 'get','CommandLine,ProcessId', '/format:list'];
const stdout = await promisify(childProcess.execFile)('wmic', args, {maxBuffer: TEN_MEBIBYTE});
```

Reply:
Yes

Reply:
updated